Resources/input/fsh/ISiKMedizinischeBehandlungseinheit.fsh

```diff
@@ -5,15 +5,17 @@ Id: ISiKMedizinischeBehandlungseinheit
 * active 1..1 MS
 * type 1.. MS
 * specialty 1.. MS
+  * ^comment = "Kodierung alle Fachbereiche unter die die Behandlungseinheit fällt."
+* specialty.coding 1.. MS
   * ^slicing.discriminator.type = #pattern
   * ^slicing.discriminator.path = "$this"
   * ^slicing.rules = #open
-* specialty contains 
+* specialty.coding contains 
   Fachrichtung 1..1 MS and
```

Comment:
question/issue: Wirklich nur 1 Fachrichtung Codierbar für das Profil Behandlungseinheit?
todo: hier in jedem Fal Motivation für Kardinalität hinzu (mit Verweis auch das "Design" von ISIKMedizinischeBehandlungseinheit